### ISSUE TITLE
[Accordions-MacOS-VoiceOver]  The expanded/collapsed state not announced by VoiceOver on Firefox

### ISSUE THREAD

**Issue by Pancham97** [w3c/aria-practices#1496]
I am trying out VoiceOver on Firefox for macOS and when I navigate through the [accordions demo](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html). The expanded/collapsed state is not announced by VoiceOver. However, this is supported on other browsers like Chrome and Safari.

Could you please let me know why this might be
happening?

Firefox version: 79.0 (64-bit)
VoiceOver version: v10

Thanks,

**Comment by mcking65** [comment]
@Pancham97 

Sorry for the long tardy response.

This is most likely a Firefox bug. On macOS, Chrome and Safari are the most commonly used browsers with VoiceOver because they have more comprehensive ARIA support.
